src/test/java/POM/LogInPage.java

```diff
@@ -0,0 +1,66 @@
+package POM;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class LoginPage extends BasePage{
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it.

![image](https://github.com/CostikM/A53-QA-Automation-Framework/assets/104761482/fc5389dd-0f18-4743-94cd-5f5644307b35)